semgrep-ci/github/onboard-repos.sh

```diff
@@ -0,0 +1,83 @@
+#!/opt/homebrew/bin/bash
+
+# TODO: Work out a way to change the cron per-repo (probably not critical but wd be better)
+# Where does the secret come from? Maybe we should do the reusable workflows?
+
+if [[ -z $1 ]]
+then
+  echo "You must set an org name"
+  exit 1
+fi
+
+if ! [[ -f semgrep.yml ]]
+then
+  echo "Please create a semgrep.yml in the current directory to add to every repo."
+  exit 1
+fi
+
+# Set up variables
+GH_ORG_NAME=$1
+PR_TITLE="Add semgrep.yml to .github/workflows to run Semgrep scans"
+PR_BODY="Adding Semgrep GitHub action to scan code for security issues"
+REPO_LIMIT=5
```

Comment:
Let's maybe increase from 5 and add a comment to let the user know to change this value

Reply:
Absolutely! That was a testing artifact 🙈